CMakeLists.txt

```diff
@@ -0,0 +1,132 @@
+cmake_minimum_required(VERSION 3.8.2)
+project(ral2-sample-driver)
+
+# Documentation
+if(CMAKE_BUILD_DOCS)
+  add_subdirectory(docs)
+  return()
+endif()
+
+# Forge/READY Robotics CMake library dependencies
+set(RR_CATKIN_DEPS
+  device-abstraction-layer
+  device-tools
+  package-manager-utils
+  robot-abstraction-layer
+  robot-abstraction-layer-utils
+  forge-core-logger-cpp
+  forge-core-storage
+  forge-core-common
+  shared-library-cpp
+  ral-messages
+  ral2-driver
+  robot-configuration
+  robot-viz-tools
```

Comment:
What is robot-viz-tools required for?

Reply:
I'm not sure it is. I copied the doosan, which copied the kawasaki which copied the....

Reply:
Far enough, lol

Reply:
I'll remove it and see if it still compiles/runs without it.

Reply:
Addressed.